cmd/bootstrap/cmd/block.go

```diff
@@ -26,8 +26,12 @@ func parseChainID(chainID string) flow.ChainID {
 		return flow.Mainnet
 	case "test":
 		return flow.Testnet
-	case "emulator":
-		return flow.Emulator
+	case "canary":
```

Comment:
not 100% related to this PR, but this has a side effect on external references to addresses (e.g. loader, simulator...) when changed on other networks

Reply:
What is the side effect? The address generator doesn't necessarily change - only the chain ID.

Reply:
isn't the address generator dependent on the chainID as input?

Reply:
It is, but it's a surjective mapping from chain ID to address space. Eg. `testnet` and `canary` have different chain IDs, but the same address space. See https://github.com/onflow/flow-go/blob/72f86d3549b10e3139cc566bc3697880482d4642/model/flow/chain.go#L40-L53